docs/beyond_basics/101-148-clonepriority.rst

```diff
@@ -0,0 +1,97 @@
+In order to be able to retrieve subdataset *data* from the subdatasets after cloning the hierarchy of datasets, the RIA store with the storage special remote needs to be configured as a clone candidate.
+Importantly, it should not only be configured as one alternative, but it should be configured as the first location to try to clone from -- else, cloning from the wrong RIA store could succeed and prevent any configured second clone candidate location from being tried.
+
+.. note::
+
+   The most likely use case for such a scenario is in the case of centrally managed data with data administrators that provide and manage the data for its users.
+
+The priority of subdataset clone locations is configured by attaching a *cost* to a source candidate ``<name>``.
+The cost is a three digit value (range ``000-999``), and the lower the cost of a candidate, the higher its priority, i.e., the candidate with the lowest cost is attempted first.
+In order to prefer any particular RIA store for subdataset cloning, one could configure the superdataset with the following command [#f3]_::
+
+    $ git config -f .datalad/config datalad.get.subdataset-source-candidate-000mypreferredRIAstore ria+http://store.datalad.org#{id}
+
+where ``mypreferredRIAstore`` is the (arbitrary) ``<name>`` of the source candidate, and the ``000`` prefix is the (lowest possible) cost.
+Such a configuration will ensure that the first location any subdataset is attempted to be installed from is the RIA store at ``store.datalad.org``.
+Only if the dataset is not found in there under its ID, other sources are tried.
+Note that in the case where no cost is provided together with the candidate name, a default cost of ``700`` is used.
+
+.. findoutmore:: What are the "default" costs for preexisting clone candidates?
+
+   The following list provides and overview of which locations are attempted for cloning and their associated costs:
+
+    - ``500`` for the superdatasets' remote URL + submodule path
+    - ``600`` for the configured submodule URL in ``.gitmodules``
+    - ``700`` for any unprioritized ``datalad.get.subdataset-source-candidate`` config
+    - ``900`` for the local subdataset path
+
+
+With the ``datalad.get.subdataset-source-candidate`` configuration any number of (differently named) clone candidates can be set and prioritized.
+This allows precise access control over subdataset clone locations, and can -- depending on how many subdataset locations are known and functional -- speed up dataset installation.
+
+.. note::
+
+   Do note how the clone candidate configuration did not specify a concrete dataset in the RIA store, but used the ``{id}`` placeholder, which will expand to the subdataset's :term:`dataset ID` upon cloning.
```

Comment:
Maybe add some additional info on what placeholders are available. I am pasting the relevant description from the docstring below:

```
    A template string assigned to such a variable can utilize the Python format
    mini language and may reference a number of properties that are inferred
    from the parent dataset's knowledge about the target subdataset. Properties
    include any submodule property specified in the respective `.gitmodules`
    record. For convenience, an existing `datalad-id` record is made available
    under the shortened name `id`.
```

We are likely to extend the list of properties in the future.